dvc/fs/webhdfs.py

```diff
@@ -31,17 +31,22 @@ def _get_kwargs_from_urls(urlpath):
         )
 
     def _prepare_credentials(self, **config):
-        if "webhdfs_token" in config:
-            config["token"] = config.pop("webhdfs_token")
-
+        self._ssl_verify = config.pop("ssl_verify", True)
+        config["kerb_kwargs"] = {}
+        if "kerberos_principal" in config:
+            config["kerb_kwargs"]["principal"] = config.pop(
+                "kerberos_principal"
+            )
         return config
 
     @wrap_prop(threading.Lock())
     @cached_property
     def fs(self):
         from fsspec.implementations.webhdfs import WebHDFS
 
-        return WebHDFS(**self.fs_args)
+        fs = WebHDFS(**self.fs_args)
+        fs.session.verify = self._ssl_verify
```

Comment:
Is this one the same as using `use_https`?

Reply:
In the long term, it might be worth adding an option for `WebHDFS` class to pass some args to the session it creates.

Reply:
I think it also might be useful to allow the caller to pass in its own session object, long term. Something like what pylivy does for example:
https://github.com/acroz/pylivy/blob/master/livy/session.py#L112